daml/Marketplace/Clearing/Role.daml

```diff
@@ -79,11 +79,14 @@ template Offer
     signatory operator
 
     controller provider can
-      Accept : ContractId Role
+      Accept : (ContractId Role, ContractId Clearing.Service, ContractId Market.Service)
         with
           ccpAccount : Account
         do
-          createOrLookup Role with ..
+          clearingCid    <- createOrLookup Clearing.Service with customer = provider, clearingAccount = ccpAccount, marginAccount = ccpAccount, ..
```

Comment:
Do we need another account in order to do this? Or can the clearing role offer itself a service with the ccp account as both?